Tests/Kernel/Symfony14KernelTest.php

```diff
@@ -119,5 +117,24 @@ public function testShouldCopyLegacyResponseData()
         $this->assertEquals($headers['Location'], $response->headers->get('Location'));
         $this->assertEquals($headers['X-Custom-Header'], $response->headers->get('X-Custom-Header'));
     }
+    
+    protected function createProphesizedContainer($request)
+    {
+        $container = $this->prophesize('Symfony\Component\DependencyInjection\ContainerInterface');
+
+        if (Kernel::MAJOR_VERSION === 2) {
```

Comment:
The code in this condition is the same as in the ```else```. Can you remove it and change the ```elseif``` to an ```if````